### DISCUSSION TITLE
rustdesk makes big /tmp files, how to solve it?

### DISCUSSION THREAD

**Discussion by sakura-umi** [rustdesk/rustdesk#614] [Q&A]
I am using rustdesk server on a Linux server, but recently I find that the directory /tmp is too big (50G) due to many files whose name is "/tmp/RustDesk_{some numbers}". What's the problem about this? How can I solve this? Thank you.

**Comment by rustdesk** [comment]
I posted it on the discord to check if anyone else encountered the same issue.

**Reply by fufesou** [reply]
[run_cmds](https://github.com/rustdesk/rustdesk/blob/f7d2ec0899c39fd66e4dd66517ed2139a6db97c5/src/platform/linux.rs#L608) will create tmp files. It may not be a good way. I'll change [the method](https://github.com/fufesou/rustdesk/blob/e477b0d951ffaafb800c37fb624cfee3133f90ce/src/platform/linux.rs#L425) in the next PR soon.

**Reply by ybyygu** [reply]
I confirmed this issue. rustdesk created too many temp files in /tmp

`$ ls -l RustDesk_*5 | head -n 10
-rw-r--r-- 1 root root 135 Dec 30 11:47 RustDesk_1672372022415
-rw-r--r-- 1 root root  27 Dec 30 11:46 RustDesk_1672372009915
-rw-r--r-- 1 root root  27 Dec 30 11:46 RustDesk_1672372004415
-rw-r--r-- 1 root root  27 Dec 30 11:46 RustDesk_1672372001345
-rw-r--r-- 1 root root  27 Dec 30 11:46 RustDesk_1672371999465
-rw-r--r-- 1 root root  27 Dec 30 11:46 RustDesk_1672371997625
-rw-r--r-- 1 root root  27 Dec 30 11:46 RustDesk_1672371989675
-rw-r--r-- 1 root root  27 Dec 30 11:46 RustDesk_1672371985395
-rw-r--r-- 1 root root  27 Dec 30 11:46 RustDesk_1672371984165
-rw-r--r-- 1 root root  27 Dec 30 11:46 RustDesk_1672371975665

$ cat RustDesk_1672372022415
ps -u 120 -o pid= | xargs -I__ cat /proc/__/environ 2>/dev/null | tr '\0' '\n' | grep '^XAUTHORITY=' | tail -1 | sed 's/XAUTHORITY=//g'%`

The version I installed: 1.1.8

**Reply by fufesou** [reply]
try 1.1.9 or nightly please
I fix this issue about May 25.

**Reply by ybyygu** [reply]
Thank you for your reply! I installed the new version. It works!
